.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Description
+
+Fixes #<ISSUE-NUMBER>
+
+Note: It's a good idea to open an issue first for discussion.
+
+## Checklist
+- [ ] **Tests** pass
+- [ ] **Lint** pass: `bundle exec rubocop -a`
```

Comment:
I don't want to recommend `-a` as a default, as it can make things weird sometimes.

Reply:
sounds good, fixed.